layouts/blog/single.html

```diff
@@ -1,37 +1,70 @@
-        <p>{{ .Params.description }}</p>
-      </div>
-      <p class="post-author">{{ (.Params.publishdate).Format "Jan 2, 2006" }} <span> | </span> By {{ replace (replace .Params.attribution "(" "- ") ")" "" }}</p>
-      {{ if .Page.Params.target_release }}
-          {{ $page_parts := split .Page.Params.target_release "." }}
-          {{ $page_version := int (index $page_parts 0) }}
-          {{ $page_revision := int (index $page_parts 1) }}
-          {{ $page_normalized_version := add (add (mul $page_version 1000000) (mul $page_revision 1000)) 0 }}
+<article class="post" itemscope itemtype="http://schema.org/BlogPosting">
+  <div class="header-content">
+    <h1>{{ .Params.title }}</h1>
+    <p>{{ .Params.description }}</p>
+  </div>
+  <p class="post-author">
+    {{ (.Params.publishdate).Format "Jan 2, 2006" }} <span> | </span> By {{
+    replace (replace .Params.attribution "(" "- ") ")" "" }}
+  </p>
+   {{ if .Page.Params.target_release }}
+      {{ $page_parts := split .Page.Params.target_release "." }}
+      {{ $page_version := int (index $page_parts 0) }}
+      {{ $page_revision := int (index $page_parts 1) }}
+      {{ $page_normalized_version := add (add (mul $page_version 1000000) (mul $page_revision 1000)) 0 }}
 
-          {{ $site_parts := split .Site.Data.args.version "." }}
-          {{ $site_version := int (index $site_parts 0) }}
-          {{ $site_revision := int (index $site_parts 1) }}
-          {{ $site_normalized_version := add (add (mul $site_version 1000000) (mul $site_revision 1000)) 0 }}
+      {{ $site_parts := split .Site.Data.args.version "." }}
+      {{ $site_version := int (index $site_parts 0) }}
+      {{ $site_revision := int (index $site_parts 1) }}
+      {{ $site_normalized_version := add (add (mul $site_version 1000000) (mul $site_revision 1000)) 0 }}
 
-          {{ if gt $site_normalized_version $page_normalized_version }}
-              <div>
-                  <aside class="callout warning">
-                      <div class="type">
-                          {{- partial "large_icon.html" "callout-warning" -}}
-                      </div>
-                      <div class="content">
-                          {{ printf (i18n "target_release") .Page.Params.target_release }}
-                      </div>
-                  </aside>
-              </div>
-          {{ end }}
-      {{ end }}
-      <div>
-        {{ .Content }}
+      {{ if gt $site_normalized_version $page_normalized_version }}
+  <div>
+    <aside class="callout warning">
+      <div class="type">
+        {{- partial "large_icon.html" "callout-warning" -}}
+      </div>
+      <div class="content">
+        {{ printf (i18n "target_release") .Page.Params.target_release }}
       </div>
-      {{ partial "posts_navigation.html" . }}
-    </article>
-{{ end }}
\ No newline at end of file
+    </aside>
+  </div>
+  {{ end }} {{ end }}
+  <div>{{ .Content }}</div>
+    <p class="social-share__title"><h4>SHARE</h4></p>
```

Comment:
```suggestion
    <p><h4>Share this blog on social!</h4></p>
```

Reply:
the style class `social-share__title` doesn't exist, so doesn't do anything here.

Reply:
Can we fix the title here? `SHARE` looks out of place.